.storybook/helpers/files.data.ts

```diff
@@ -50,15 +70,29 @@ export const filesRoot: File[] = [
     ],
   },
   {
+    disabled: true,
     name: 'package.json',
   },
   {
     name: 'tsconfig.json',
   },
 ];
 
-export function getFileTreeNodeMarkup(files: File[] = filesRoot) {
+export function getFileTreeNodeMarkup(
+  files: File[] = filesRoot,
+  args: { asLink?: boolean; clrSelected?: boolean } = {}
+) {
   return files
-    .map(file => `<clr-tree-node>${file.name}${file.files ? getFileTreeNodeMarkup(file.files) : ''}</clr-tree-node>`)
+    .map(
+      file => `
+        <clr-tree-node
+        [clrDisabled]="${file.disabled}"
+        [clrExpanded]="${file.expanded}"
+        ${args.clrSelected !== undefined && file.selected !== undefined ? `[clrSelected]="${file.selected}"` : ''}
+        >
+            ${args.asLink ? `<a href="javascript:;" class="clr-treenode-link">${file.name}</a>` : file.name}
+          ${file.files ? getFileTreeNodeMarkup(file.files, args) : ''}
+        </clr-tree-node>`
```

Comment:
```suggestion
      file => `
        <clr-tree-node
          [clrDisabled]="${file.disabled}"
          [clrExpanded]="${file.expanded}"
          ${args.clrSelected !== undefined && file.selected !== undefined ? `[clrSelected]="${file.selected}"` : ''}
        >
          ${args.asLink ? `<a href="javascript:;" class="clr-treenode-link">${file.name}</a>` : file.name}
          ${file.files ? getFileTreeNodeMarkup(file.files, args) : ''}
        </clr-tree-node>`
```

indentation is bit off